css/style.css

```diff
@@ -2,21 +2,23 @@
   box-sizing: border-box;
 }
 
-@media (max-width: 376px) {
-  html,
+html,
   body {
+    margin: 0;
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/GurevAleksey_itogovaya/pullRequest?prid=6183649)